src/lib/Config.ts

```diff
@@ -14,12 +14,16 @@
  *  limitations under the License.
  */
 
-const {promisify} = require('util');
-const fs = require('fs');
-const homedir = require('os').homedir();
-const path = require('path');
-const fileExists = promisify(fs.exists);
+import {promisify} from 'util';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as os from 'os';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
Why are we doing this?

Reply:
It's a bit of a leftover. Changed to a proper import.